rfcs/text/0428-extending-types.md

```diff
@@ -0,0 +1,92 @@
+- Feature Name: extending-types
+- Start Date: 2022-07-11
+- RFC PR: [blockprotocol/blockprotocol#428](https://github.com/blockprotocol/blockprotocol/pull/428)
+- Block Protocol Discussion: [blockprotocol/blockprotocol#0000](https://github.com/blockprotocol/blockprotocol/discussions/0000)
+
+# Summary
+
+[summary]: #summary
+
+As a follow-up to the [Graph type system RFC](./0352-graph-type-system.md), this RFC will describe the behavior of which types in the type system can be extended and duplicated.
+
+# Motivation
+
+[motivation]: #motivation
+
+Why are we doing this? What use cases does it support? What is the expected outcome?
+
+# Guide-level explanation
+
+[guide-level-explanation]: #guide-level-explanation
+
+Explain the proposal as if it was already included in the protocol and you were teaching it to another Block Protocol implementor. That generally means:
```

Comment:
```suggestion
Explain the proposal as if it was already included in the protocol and you were teaching it to another Block Protocol implementer. That generally means:
```

So says the ngram.
- [StackExchange answer](https://english.stackexchange.com/questions/195699/implementor-vs-implementer)
- [ngram result](https://books.google.com/ngrams/graph?content=implementer%2Cimplementor&year_start=1929&year_end=2019&corpus=26&smoothing=0&case_insensitive=true&direct_url=t4%3B%2Cimplementer%3B%2Cc0%3B%2Cs0%3B%3Bimplementer%3B%2Cc0%3B%3BImplementer%3B%2Cc0%3B%3BIMPLEMENTER%3B%2Cc0%3B.t4%3B%2Cimplementor%3B%2Cc0%3B%2Cs0%3B%3Bimplementor%3B%2Cc0%3B%3BImplementor%3B%2Cc0%3B%3BIMPLEMENTOR%3B%2Cc0)

Reply:
That's my mistake I think, unless I copied it from somewhere else 😁